diff --git a/node_modules/three/examples/jsm/loaders/FBXLoader.js b/node_modules/three/examples/jsm/loaders/FBXLoader.js
index bc2d837..d49bf9f 100644
--- a/node_modules/three/examples/jsm/loaders/FBXLoader.js
+++ b/node_modules/three/examples/jsm/loaders/FBXLoader.js
@@ -43,7 +43,7 @@ import {
 	SRGBColorSpace,
 	ShapeUtils
 } from 'three';
-import * as fflate from '../libs/fflate.module.js';
+import * as fflate from 'fflate';
 import { NURBSCurve } from '../curves/NURBSCurve.js';
 
 /**
